astro.config.mjs

```diff
@@ -94,26 +94,21 @@ export default defineConfig({
         { label: 'Reference',
           // translations: { 'fr': 'Référence', },
           items: [
-            { label: 'Tidal functions',
-              autogenerate: { directory: 'reference/tidal' },
-              collapsed: true,
-              badge: 'WIP',
+            { label: 'Tidal',
+              items: [
+                {label: 'Configuration | The boot file', link: 'reference/tidal/configuration' },
+                {label: 'Functions', autogenerate: { directory: 'reference/tidal/functions' }, badge: 'WIP', collapsed: true},
+                {label: 'Mini-notation', autogenerate: {directory: 'reference/tidal/mini-notation' }, collapsed: true},
+              ],
             },
             { label: 'SuperDirt',
-              autogenerate: { directory: 'reference/superdirt' },
-              collapsed: true,
-            },
-            { label: 'Configuration',
               items: [
-                { label: 'Tidal', link: '/reference/config/tidal' },
-                { label: 'SuperDirt', link: '/reference/config/superdirt' },
-              ]
-            },
-            { label: 'Mini-notation',
-              autogenerate: { directory: '/reference/mini-notation' },
+                {label: 'Configuration | The startup file', link: 'reference/superdirt/configuration'},
+                {label: 'Default assets', autogenerate: { directory: 'reference/superdirt/default-assets' }, collapsed: true},
```

Comment:
Not sure if "assets" is the right word for synths and effects ... but I can't think of anything else. "Default components" maybe? But "assets" works anyway.

Reply:
How about 'default library'?